.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: CI
+
+# About email notifications, see <https://git.io/JUo27>.
```

Comment:
That is a long document covering GH email notifications in general. I don't understand what this is supposed to tell me here.

Reply:
like this part in travis.yml
https://github.com/rust-lang/miri/blob/aa832c1133217a079ae5d9af65dd0b7ad96aca07/.travis.yml#L52-L57

In short, you receive mails for build failure.

Reply:
We mostly rely on bors for this nowadays (these notifications predate us using bors)... so I'd say let's go with whatever a reasonable default is and we can always tweak it later.

Reply:
I'll remove it then.